packages/shared/tsup.config.ts

```diff
@@ -1,7 +1,7 @@
 import { defineConfig } from 'tsup';
 
 export default defineConfig((options) => ({
-  entry: ['src/index.ts'],
+  entry: ['src/index.ts', 'src/archive-storybook/scripts.ts'],
```

Comment:
Is there any reason not to build the main.js too?

Reply:
There were errors loading SB with the built version, but it was late so I didn't dig in. Will try that again.

Reply:
Building the main file is working now!